haas/cli.py

```diff
@@ -89,7 +92,14 @@ def do_delete(url):
     return check_status_code(requests.delete(url))
 
 @cmd
-def serve():
+def serve(port):
+    try:
+        port = schema.And(schema.Use(int), lambda n: MIN_PORT_NUMBER <= n <= MAX_PORT_NUMBER).validate(port)
+    except schema.SchemaError:
+	sys.exit('Error: Invaid port. Must be in the range 1-65535.')
+    except as e:
```

Comment:
You need a type here; `except Exception as e` is probably what you right. As it stands, this is a syntax error.